### ISSUE TITLE
map rotation `if` action does not support map argument

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#1142]
According to [wiki](https://wiki.unvanquished.net/wiki/Server/Map_Rotation), `if` can both load map or labels, and this map rotation must be doable:

```python
antares
if numClients > 8 parpax
if numClients < 4 parpax-mini
chasm
forlorn
```

But only labels are supported, so to get the same behavior, this over-complicated rotation has to be done instead:

```python
antares
if numClients > 8 #parpax
if numClients < 4 #parpax-mini
#parpax
parpax
#parpax-mini
parpax-mini
goto chasm
#chasm
chasm
forlorn
```

**Comment by Gireen** [comment]
It seems that maps are treated like goto statements.
The first rotation results in following error
 `Warn: label, map, or rotation parpax-mini not found in rotation1`
if you add that map to the rotation it will jump there without label.

**Comment by illwieckz** [comment]
fixed by #1145
